src/Sentry/Internal/StackFrame.cs

```diff
@@ -114,7 +114,15 @@ public RealStackFrame(StackFrame frame)
 
     [UnconditionalSuppressMessage("Trimming", "IL2026:Members annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code", Justification = AotHelper.SuppressionJustification)]
     public MethodBase? GetMethod() => AotHelper.IsNativeAot
+#if !NET8_0_OR_GREATER
+#pragma warning disable CS0162 // Unreachable code detected
+#endif
+        // Only unreachable outside NET8_0_OR_GREATER
+        // ReSharper disable once HeuristicUnreachableCode
         ? null
+#if !NET8_0_OR_GREATER
+#pragma warning restore CS0162 // Unreachable code detected
+#endif
         : _frame.GetMethod();
```

Comment:
How about the following? It's a bit more readable to me:

```suggestion
    public MethodBase? GetMethod() 
#if NET8_0_OR_GREATER
				=> AotHelper.IsNativeAot ? null : _frame.GetMethod();
#else
				=> _frame.GetMethod();
#endif
```